src/views/DatasetManager/components/CreateDatasetButton/components/DatasetCreationParameters/DatasetCreationParameters.js

```diff
@@ -183,36 +177,45 @@ export const DatasetCreationParameters = ({ dataSourceRunTemplates, parentDatase
         };
   }, [t, isSubDatasetCreationWizard]);
 
+  const sourceTypeComponent =
+    selectedRunner && Object.keys(selectedRunner).length > 0 ? (
```

Comment:
Maybe we could create a bool constant with an easy-to-read name to replace both this expression `selectedRunner && Object.keys(selectedRunner).length > 0` and its opposite `!selectedRunner || Object.keys(selectedRunner)?.length === 0` that is used in the `return` statement